src/Internal/Requirements/RequirementValidator.php

```diff
@@ -39,6 +39,14 @@ public static function instance(): RequirementValidator {
 	 * @param RuntimeException $e
 	 */
 	protected function add_admin_notice( RuntimeException $e ) {
+		// Check if the plugin is active. If not, display error message and terminate.
+		if ( ! is_plugin_active( 'woocommerce-google-listings-and-ads/woocommerce-google-listings-and-ads.php' ) ) {
+			die(
+				esc_html( $e->getMessage() )
```

Comment:
For wp-cli we might want to add a PHP_EOL here otherwise when I activate on the command line it doesn't jump to the next line before showing the prompt.

Reply:
Thanks, @mikkamp, I have appended PHP_EOL to the string.